test/test_base58.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2018 John Newbery
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""Test base58.py"""
+import unittest
```

Comment:
Please sort according to pep8 (std library first, then third party libraries, then package modules). This would become:

```python
from binascii import unhexlify
import unittest

import base58
```